### ISSUE TITLE
Add strings to existing text, using Xpath

### ISSUE THREAD

**Issue by Pierre-de-la-Verre** [microsoft/XmlNotepad#256]
(Seem to be similar to  #53 )

I want to 
- search for a text with Xpath and
- modify the found text with a new string as prefix, e.g. "hello "<found text>

Searching would be done with Xpath, e.g. with "/PLAY/ACT/SCENE[1]/SPEECH[1]/SPEAKER/text()" for Hamlet.xml. 
But how to add the prefix to the found text?

**Comment by lovettchris** [comment]
Yep, this is an extension of https://github.com/microsoft/XmlNotepad/issues/53
I agree this would be handy.
